src/deluge/gui/views/automation_instrument_clip_view.h

```diff
@@ -120,10 +121,8 @@ class AutomationInstrumentClipView final : public ClipView, public InstrumentCli
 	void renderAutomationEditor(ModelStackWithTimelineCounter* modelStack, InstrumentClip* clip, Instrument* instrument,
 	                            uint8_t* image, uint8_t occupancyMask[], int32_t renderWidth, int32_t xScroll,
 	                            uint32_t xZoom, int32_t yDisplay = 0, bool drawUndefinedArea = true);
-	void renderRow(ModelStackWithAutoParam* modelStack, uint8_t* image, uint8_t occupancyMask[], bool, uint32_t,
```

Comment:
simplified renderRow function definition. Not all the parameters were being used. Only defined what's needed